rhombus/scribblings/ref-stxobj.scrbl

```diff
@@ -760,7 +758,7 @@ Metadata for a syntax object can include a source location and the raw
 
 
 @doc(
-  fun Syntax.unwrap_group(stx :: Group) :: List.of(Syntax)
+  fun Syntax.unwrap_group(stx :: Group) :: Listable.of(Syntax)
```

Comment:
These *result* annotations should simply say `List.of`.

Reply:
HF